app/boards/arm/preonic/preonic_rev3.keymap

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright (c) 2020 The ZMK Contributors
+//
+// SPDX-License-Identifier: MIT
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+
+#define DEFAULT 0
+#define LOWER   1
+#define RAISE   2
+#define ADJ     3
+
+/ {
+  chosen { zmk,matrix_transform = &layout_grid_transform; }; 
```

Comment:
Ditto, some indent issues.

Reply:
Corrected all the formatting issues in this file.